lib/huginn_acumen_product_agent/concerns/inv_product_query_concern.rb

```diff
@@ -46,6 +46,8 @@ def process_inv_product_response(raw_data, digital_format_list)
         product['isTaxable'] = get_field_value(p, 'Inv_Product.Taxable') == '1'
         product['productAvailability'] = get_field_value(p, 'Inv_Product.Not_On_Website') == '0'
         product['noBackorderFill'] = get_field_value(p, 'Inv_Product.No_Backorder_Fill') == '1'
+        product['nonInventory'] = get_field_value(p, 'Inv_Product.Non_Inventory') == '1'
+        product['assembly'] = get_field_value(p, 'Inv_Product.Assembly') == '1'
```

Comment:
@Jhalter5Stones I would roll this into a single field. We only use them in the context of inventory tracking,

`product['trackInventory'] = _____ || _____`.

Also, you need to update the Acumen query to actually _fetch_ these fields.

Reply:
@mrogers-5s Fixed.